mllib/src/test/scala/org/apache/spark/mllib/fpm/PrefixSpanSuite.scala

```diff
@@ -107,6 +107,75 @@ class PrefixSpanSuite extends SparkFunSuite with MLlibTestSparkContext {
       (Array(4, 5), 2L),
       (Array(5), 3L)
     )
-    assert(compareResult(expectedValue3, result3.collect()))
+    assert(compareResult(expectedValue3.map(x => (x._1.map(Array(_)), x._2)), result3.collect()))
+
+    val sequences4 = Array(
```

Comment:
Is this a test for non-temporal sequences? If so, please move to a separate `test` block